app/views/apps/index.html.erb

```diff
@@ -9,9 +9,7 @@
   <div class="grid grid-flow-col sm:auto-cols-max justify-start sm:justify-end gap-2">
     <%= authz_link_to :green, new_app_path do %>
       <%= inline_svg("create_plus.svg", classname: "w-4 fill-current opacity-50 shrink-0") %>
-      <span class="hidden xs:block ml-2">
-        Add a new app
-      </span>
+      <span class="hidden xs:block ml-2">Add a new app</span>
     <% end %>
```

Comment:
I think we should remove this change, it will cause an unnecessary conflict with the page component PR.